bodhi/server/models.py

```diff
@@ -62,6 +46,167 @@
     log.warning("Could not import 'rpm'")
 
 
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+
+
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# http://techspot.zzzeek.org/2011/01/14/the-enum-recipe
+
+import re
+
+from sqlalchemy.types import SchemaType, TypeDecorator, Enum
```

Comment:
The idea is to keep all the import statements together. https://github.com/ankit01ojha/bodhi/blob/b7fdff2e6f66d8a487848f8424189bec798a55c9/bodhi/server/models.py#L1-L83
Please check the import guide https://www.python.org/dev/peps/pep-0008/#imports